### PR TITLE
Split reporting dag

### PR DESCRIPTION
**Pull request by cbrianbet**
Split linelist and aggregates into different Dags

### CODE REVIEW

File: production/dags/reporting_aggregates_tables_etl_dag.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+load_aggregate_hts_pns_sexualpartner >> load_aggregate_hts_pnschildren >> load_aggregate_hts_tbscreening >> load_aggregate_hts_teststrategy  >> load_aggregate_hts_uptake >> load_aggregate_iit_tracing_status >> load_aggregate_nupi >> load_aggregate_optimize_current_regimens  >> load_aggregate_optimize_start_regimens  >> load_aggregate_prep_cascade
+load_aggregate_prep_cascade >> load_aggregate_recencyuploads >> load_aggregate_time_to_art_grp  >> load_aggregate_time_to_art_last_12m >> load_aggregate_time_to_art  >> load_aggregate_time_to_first_vl_grp >> load_aggregate_time_to_vl_12m >> load_aggregate_otz  >> load_aggregate_TPT  >> load_aggregate_treatmentoutcomes >> load_aggregate_txcurr
+load_aggregate_txcurr >> load_aggregate_tx_new_hts_cascade  >> load_aggregate_vl_outcome_and_uptake >> load_aggregate_otz_eligibility_and_enrollments >> load_aggregate_otz_outcome >> load_aggregate_ovc_count >> load_aggregate_prep_STIOutcomes >> load_aggregate_prep_discontinuations >> load_aggregate_prep_testing_at_1month_refill >> load_aggregate_prep_testing_at_3month_refill
```
Comment (nthusi-codes) [inline production/dags/reporting_aggregates_tables_etl_dag.py:125]: Just a note. When moving on to a new line in the dag task routes, the last task in the line should also be the first task in the next line. i.e 
....... >> load_a_task >> load_b_task
load_b_task >> load_c_task ........

**Review by nthusi-codes — CHANGES_REQUESTED** [review]

File: production/dags/reporting_aggregates_tables_etl_dag.py (hunk 1 above)
Comment (cbrianbet) [reply production/dags/reporting_aggregates_tables_etl_dag.py:125]: Hey @nthusi-codes I see that this is the way I have written the code 
ie last task on line 121 is the first on line 122 and so on

File: production/dags/reporting_tables_etl_dag.py [hunk 2]
```diff
@@ -147,12 +53,11 @@
...
-load_linelist_vl_non_suppressed >> load_aggregate_otz_eligibility_and_enrollments >> load_aggregate_otz_outcome >> load_aggregate_ovc_count >> load_aggregate_prep_STIOutcomes >> load_aggregate_prep_discontinuations >> load_aggregate_prep_testing_at_1month_refill >> load_aggregate_prep_testing_at_3month_refill
+load_linelist_FACTART >> load_linelist_FACTART_Palantir >> load_linelist_vl_non_suppressed >> load_all_emr_sites  >> load_linelist_adverse_events >> load_linelist_appointments >> load_linelist_covid  >> load_linelist_hts_risk_categorization_and_test_results >> load_linelist_otz_eligibility_and_enrollments >> load_linelist_otz >> load_linelist_ovc_eligibility_and_enrollment >> load_linelist_ovc_enrollment >> load_linelist_Prep >> load_linelist_trans_hts >> load_linelist_trans_pns >> load_linelist_viralload >> reporting_aggregates_etl_trigger
```
[the remark this answers is not in the document]
Comment (cbrianbet) [reply production/dags/reporting_tables_etl_dag.py:63]: Hi @nthusi-codes all the tasks or on one line here I dont think it applies

File: production/dags/reporting_aggregates_tables_etl_dag.py (hunk 1 above)
Comment (nthusi-codes) [reply production/dags/reporting_aggregates_tables_etl_dag.py:125]: Seems like I missed that

**Review by nthusi-codes — APPROVED** [review]

**Pull request merged**